### PR TITLE
adding optimize doc explaining our interface

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
If this does not give you a full enough picture please let me know.

Some confusing parts to focus on to be sure they are understandable:
-maxiter
-maxfeval

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]
Great job!

File: docs/workflow operations/Optimize Doc.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+## Goal
+Used to find the minimal or maximum value to a given parameter value or time that a QoI (Quality of Interest) constraint does not exceed the provided threshold.
+
+We can ask questions to try to find out a good start time for an intervention:
+
+I want to turn on "masking" (beta set to a smaller value ~0.1) 
+but I want to know how late I can do it while making the number of infected never exceed a specified number.
+
+We can use questions to try to find out parameter values:
+I want to turn on "quarantine" in day 30. How strong/effective (high beta needs to be) will this quarantine need to be in order to keep the number of infected lower than a (specified threshold) number?
```
Comment (mwdchang) [inline docs/workflow operations/Optimize Doc.md:11]: Maybe rephrase here,  as well as the similar copies below. We don't have context into what `beta` refers to here, so you want to spell out things in full.

E.g. "I want to turn on quarantine on day 30 for 90 days, represented by the variable xyz, how strong/weak will xyz need to be in order to keep the number of infected below ...."

Comment (Tom-Szendrey) [reply docs/workflow operations/Optimize Doc.md:11]: Thank you that is a good point. Especially if we are going to feed this documentation through llm it is best to be explicit and provide the full scope in one place rather than assume we understand beta here is a parameter that has some impact on infected

**Comment by mwdchang** [comment]
FYI I don't know if the formatting you have in the markdown is intended to drive the rendering of the markdown, the actual render will not respect any of the indentation levels that you have created.

File: docs/workflow operations/Optimize Doc.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
+# Optimize:
+## Goal
+Used to find the minimal or maximum value to a given parameter value or time that a QoI (Quality of Interest) constraint does not exceed the provided threshold.
```
[the remark this answers is not in the document]
Comment (bigglesandginger) [reply docs/workflow operations/Optimize Doc.md]: This is a very cumbersome sentence.  Would something like this work better?

`Used to find the limit that does not exceed the threshold of a Quality of Interest constraint for a given parameter.`

File: docs/workflow operations/Optimize Doc.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+We can ask questions to try to find out a good start time for an intervention:
+
+I want to turn on "masking" (beta set to a smaller value ~0.1)
```
Comment (bigglesandginger) [inline docs/workflow operations/Optimize Doc.md:7]: If this is a quote (question being asked) it should have quotes around it, otherwise the mixing of plural and singular pronouns is just confusing.

**Review by mwdchang — APPROVED** [review]
